lib/ansible/module_utils/network/nxos/facts/l3_interfaces/l3_interfaces.py

```diff
@@ -0,0 +1,126 @@
+#
+# -*- coding: utf-8 -*-
+# Copyright 2019 Red Hat
+# GNU General Public License v3.0+
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)#!/usr/bin/python
+"""
+The nxos l3_interfaces fact class
+It is in this file the configuration is collected from the device
```

Comment:
I know that this comes out from RMB, but in any case it does sound a li'l bit awkward to me. How about reformatting this to something along the lines of `This file handles device configuration collection for a given resource. It parses and populates the facts tree based on such config'. I'll add Sandra to check it out too. Thanks!

Reply:
From a docs perspective, this is not surfaced at all (only visible within this file itself). So I suggest if you want to change things, update the RMB and it will be picked up by future modules.